iped-app/resources/scripts/tasks/RefineCategoryTask.js

```diff
@@ -148,6 +148,28 @@ function process(e){
 		}
 	}
 
+	// Calls sub-categories
+	if (mime.equals("application/x-ufed-call")) {
+		source = e.getMetadata().get("ufed:Source");
+		if (source == null) {
+			e.setCategory("Phone Calls");
+		} else {
+			source = source.toLowerCase();
+			if (source.contains("whatsapp")) {
+				e.setCategory("WhatsApp Calls");
+			} else if (source.contains("facebook")) {
+				e.setCategory("Facebook Calls");
+			} else if (source.contains("discord")) {
+				e.setCategory("Discord Calls");
+			} else if (source.contains("threema")) {
+				e.setCategory("Threema Calls");
+			} else {
```

Comment:
Shouldn't "Telegram Calls" be added in this block?

Reply:
Thanks @lfcnassif! I will fix these things you pointed out.